help/help.md

```diff
@@ -0,0 +1,132 @@
+% POSTFIX(1) Container Image Pages
+% Petr Hracek
+% September 19, 2017
+
+# NAME
+{{ spec.envvars.name }} - {{ spec.short_description }}
+
+# DESCRIPTION
+{{ spec.description }}
+
+# USAGE
+To get the memcached container image on your local system, run the following:
+
+    docker pull hub.docker.io/modularitycontainers/{{ spec.envvars.name }}
+
+Commands for testing Postfix docker container:
+
+```$ telnet localhost 25```
+
+```HELO test.localhost```
+
+```MAIL FROM:<yourname>@<domain.com>```
+
+```RCPT TO:<demo@localhost>```
+
+```DATA``` and type whatever you want
+```
+Subject: My testing docker container image
+
+Hi, Testing message
+regards
+Docker
+.
+```
+
+## How to test the postfix mail server with TLS
+
+Command for testing Postfix docker container with
+enabled TLS is ```openssl```.
+
+Telnet has not to be used because of all
```

Comment:
Fix grammar: 'Telnet has not to be used because of all' -> 'Telnet cannot be used because all'

Reply:
Also, it's inaccurate to say that _all comunication is encrypted from the beginning_. You are not using wrapper mode in the postfix config, so the client has to request TLS first. But I guess you know that, since you used 'openssl s_client -starttls smtp'.